addons/repair/models/stock_warehouse.py

```diff
@@ -63,6 +63,18 @@ def _get_production_location(self):
     def _generate_global_route_rules_values(self):
         rules = super()._generate_global_route_rules_values()
         production_location = self._get_production_location()
+
+        route = self._find_or_create_global_route('stock.route_warehouse0_mto', _('Replenish on Order (MTO)'))
+        existing_rule = route.rule_ids.search([
```

Comment:
it's not really good practice to do a search like this. I'm not sure what exactly you're trying to achieve here :thinking:

Reply:
As discussed, the idea here is a clean up to make sure the rule will be deleted when the user uninstalls the repair module.